core/src/it/java/software/amazon/awssdk/auth/InstanceProfileCredentialsProviderIntegrationTest.java

```diff
@@ -86,7 +86,7 @@ public void testNoInstanceProfiles() throws Exception {
         mockServer.setResponseFileName("sessionResponse");
         mockServer.setAvailableSecurityCredentials("");
 
-        InstanceProfileCredentialsProvider credentialsProvider = new InstanceProfileCredentialsProvider();
+        InstanceProfileCredentialsProvider credentialsProvider = InstanceProfileCredentialsProvider.create();
```

Comment:
Should we do this for our other credential providers as well?

Reply:
I think maybe it makes sense for providers that have a builder, so I've added it to `ProfileCredentialsProvider` as well.